src/module_recipe.rs

```diff
@@ -39,7 +47,7 @@ pub struct Recipe {
 impl Recipe {
     #[must_use]
     pub fn generate_tags(&self) -> Vec<String> {
-        trace!("Recipe::generate_tags()");
+        debug!("Recipe::generate_tags()");
```

Comment:
```suggestion
        trace!("Recipe::generate_tags()");
```
I would like method call logs to be trace.